jest/jest.config.js

```diff
@@ -26,5 +26,8 @@ module.exports = {
     'github-markdown-css': '<rootDir>/node_modules/identity-obj-proxy',
     '\\.(png)$': '<rootDir>/node_modules/identity-obj-proxy',
     '\\.(svg)$': '<rootDir>/jest/unit/empty.ts',
+    '@verdaccio/components/(.*)': '<rootDir>/src/components/$1',
```

Comment:
I'd like to create a common file with these aliases, but I don't know exactly where to add it...Please, any thoughts?